### PR TITLE
Make several modifications for the reader.

### PR DESCRIPTION
**Pull request by codeviking**
This modifies the base viewer, and the build, so that the output
is easier to work with in the interactive reader.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ PDF.js is community-driven and supported by Mozilla Labs. Our goal is to
 create a general-purpose, web standards-based platform for parsing and
 rendering PDFs.
 
+## Notice of Changes
```
Comment (andrewhead) [inline README.md]: Wonderful!

File: gulpfile.js [hunk 2]
```diff
@@ -798,6 +802,8 @@ gulp.task('minified-post', gulp.series('minified-pre', function (done) {
                    Terser.minify(pdfFile).code);
   fs.writeFileSync(MINIFIED_DIR + '/build/pdf.worker.min.js',
                    Terser.minify(pdfWorkerFile, optsForHugeFile).code);
+  fs.writeFileSync(MINIFIED_DIR + '/web/pdf.worker.min.js',
```
Comment (andrewhead) [inline gulpfile.js]: What's the reason for the alternative PDF worker script?

File: web/app.js [hunk 3]
```diff
@@ -1,4 +1,10 @@
-/* Copyright 2012 Mozilla Foundation
+/* NOTICE OF CHANGES:
```
Comment (andrewhead) [inline web/app.js]: Good catch!

File: gulpfile.js (hunk 2 above)
Comment (codeviking) [reply gulpfile.js]: Good question.

It just makes it easier to copy over the files into the reader project.  The worker script without this modification ends up in `build/generic/build`, this file is in `build/generic/web`.   Putting it in `build/generic/web` means everything the viewer needs is in `web`, so we can just `rsync` that directory over.

**Pull request merged**